server/README.md

```diff
@@ -0,0 +1,10 @@
+# nodejs-template
+A bash script to setup a simple template for starting projects, installing key npm packages, and running the server instantly on local port 3000 with examples
+
+## Requirements:
+The bash script will check and install necessary dependencies. If you are not on macOS or linux, you will need to install nodejs from https://nodejs.org/en/ before running.
+
```

Comment:
hi

Reply:
This issue has been resolved please recheck it!